src/v2/service/auth.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Request as ExRequest } from "express";
+
+// REDIRECT URL IS '<origin>/auth/<provider>/callback'
+const REDIRECT_URI_PATHNAME = "/callback";
+// example with discord provider:
+// REDIRECT URL IS '<origin>/auth/discord/callback'
+
+export default class AuthService {
+	rawTargetToURL(target: string) {
+		if (target === "webapp") return process.env.WEBAPP_URL;
```

Comment:
not sure how I feel about this being hardcoded, but idk what the best option would be here